app/projects/components/ContributorPathReport/index.tsx

```diff
@@ -68,7 +68,7 @@ export const IncompleteIcon = styled.span`
 export const ContributorPathReport = ({ project }: IProps) => {
   return (
     <>
-      <big>Contributors</big>
+      <big>Contributors ({project.projectMembers.length})</big>
```

Comment:
I think we should only count active members, and also be specific about it on the label: `Active Contributors #`. Or what do you think?

Otherwise this looks great!

Reply:
I agree with the part of showing only the active Contributors number in the card, however, regarding the label, if I understand what you said Jack, we can't change that label because the chart shows all contributors, not only the active, so the label shouldn't change, at least not that way, maybe we could do something like `Contributors (# active)`, that way it would be clear enough I guess.

These are my thoughts, and as Jack said, that looks great!

Reply:
Looks good to me. Regarding the conversation about "Active contributors" on the table, I agree with Juan's idea, that way we may still have the remark of how many active contributors are.